components/console_simple_init/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+
+## [1.0.1](https://github.com/espressif/esp-protocols/commits/console_simple_init-v1.0.1)
+
+### Features
+
+- Added check to allow publishing of a component without docs ([1f943690c8](https://github.com/espressif/esp-protocols/commit/1f943690c8))
```

Comment:
I would also suggest rephrasing this commit message. This a feature, but a common feature related to the `esp_protocols` repository. So this text should not appear in the release notes of the console component.

suggest using -- `fix(console): Fixed versioning, publishing and changelog generation`